app/knights/knight.py

```diff
@@ -0,0 +1,24 @@
+class Knight:
+
+    def __init__(self, config: dict) -> None:
+
+        self.name = config["name"]
+        self.power = config["power"]
+        self.hp = config["hp"]

```

Comment:
Ensure that the keys 'name', 'power', and 'hp' exist in the config dictionary to avoid KeyError. Consider using the `get` method with default values or adding error handling.